### ISSUE TITLE
When PLC is not reachable an inappropriate exception is thrown

### ISSUE THREAD

**Issue by mbalous** [killnine/s7netplus#116]
Disconnecting the PLC from power supply after `plc.Open()` is invoked produces an exception specified below:
```
The variable'DB111.DBX1.2' could not be read. Please check the syntax and try again.
```

I think this is really misleading, because a syntax correction definitely won't help. An exception with message similar to "PLC can not be reached" should be thrown in my opinion.

**Comment by Mallowan** [comment]
> An exception with message similar to "PLC can not be reached" should be thrown in my opinion.
Yes, you're right, provided that your application polls continuosly such lib properties like 'IsConnected'.
And based on this event your app disable a var polling.

Mallowan

**Comment by rapha-dev** [comment]
Right, similar to issue #112 where WrongVarFormat is returned.
Methods `Read(string)` and `Write(string, object)` should be affected. Needs better exception handling (Exception forwarding).

**Comment by rapha-dev** [comment]
Addressed in #156.
